mms/export_model.py

```diff
@@ -187,7 +189,16 @@ def generate_manifest(symbol_file, params_file, service_file, signature_file, mo
     manifest["Model"]["Description"] = model_name
     manifest["Model"]["Model-Name"] = model_name
     manifest["Model"]["Model-Format"] = "MXNet-Symbolic"
-    manifest["Engine"] = {"MXNet": 0.12}
+    
+    mxnet_version = mx.__version__
+
+    if os.path.exists(symbol_file):
+        symbol_json = json.load(open(symbol_file))
+        if MXNET_ATTRS in symbol_json:
```

Comment:
this was 'attr' in mxnet < 1.0, attr >= 1.0. we should support both flavors.

Reply:
ok, will fix that